nomad-core/src/signer.rs

```diff
@@ -57,23 +59,54 @@ impl From<AwsSigner<'static>> for Signers {
     }
 }
 
+async fn try_aws(region: &str) {
+    KMS_CLIENT.get_or_init(|| {
+        KmsClient::new_with(
+            HttpClient::new().unwrap(),
+            DefaultCredentialsProvider::new().unwrap(),
+            region.parse().expect("invalid region"),
+        )
+    });
+}
+
+async fn try_aws_with_arn(region: &str, arn: &str) {
+    let sts = StsClient::new(region.parse().expect("invalid region"));
+
+    let provider = StsAssumeRoleSessionCredentialsProvider::new(
+        sts,
+        arn.to_owned(),
+        "default".to_owned(),
+        None,
+        None,
+        None,
+        None,
```

Comment:
So we will assume operators will not configure these? Unless it becomes feature request

https://rusoto.github.io/rusoto/rusoto_sts/struct.StsAssumeRoleSessionCredentialsProvider.html